src/ages/distributions/ageOneDistribution.ts

```diff
@@ -4,9 +4,9 @@ import { AgeDistribution } from "./distributions.types";
 
 export const ageOneDistribution = async (
   age: AgeDistribution,
-  { snapshotBlock, totalEmission, finalTimestamp, initialTimestamp, id }: EpochConfig
+  { snapshotBlock, totalEmission, finalTimestamp, initialTimestamp, number }: EpochConfig
```

Comment:
I'm not a fan of `number` by the way, I find it too close to the `number` type 🤔

Reply:
any other suggestion? `epochNumber` ?

Reply:
Yes, it's a little redundant, but `epochNumber` is fine. Otherwise, `id` could be the number, and `epochName` could be `ageX-epochY` to me. `iteration` or `occurence` could work, but I'm not really fan of those. 
We can keep `number` while we don't find anything better 😄